modules/cluster-logging-upgrading-elasticsearch.adoc

```diff
@@ -168,28 +144,29 @@ green  open   audit-000001
 ----
 ====
 
-.. Verify that the log collector is updated:
+. Verify that the log collector is updated by entering the following commands and observing the output:
 +
 [source,terminal]
 ----
 $ oc get ds collector -o json | grep collector
 ----
 +
-.. Verify that the output includes a `collectort` container:
+Verify that the output includes a `collectort` container:
```

Comment:
... "collector"...

Reply:
lol I'm guessing this was a typo originally, will update